pkg/vm/vm.go

```diff
@@ -1421,6 +1415,16 @@ func (v *VM) execute(ctx *Context, op opcode.Opcode, parameter []byte) (err erro
 	return
 }
 
+func (v *VM) jump(ctx *Context, parameter []byte) {
```

Comment:
nit: would be nice to have godoc here since it's not that obvious how parameter is related to jump